### ISSUE TITLE
Installation without Optifine?

### ISSUE THREAD

**Issue by CalebDaMaker** [jrbudda/Vivecraft_118#30]
Is there anyway to install without Optifine? And please don't tell me it's needed for FPS, there are much better mods I'm trying to use that are incompatible, and I just want the answer. Thanks!

**Comment by mmaniac01** [comment]
Hi CalebDaMaker,

Vivecraft requires OptiFine for much more than just FPS. I won't pretend to know what it is required for exactly, but believe me, I've tried to run it without OptiFine and it flat out does not work.

However, if you share what mods are incompatible, or check [Vivecraft's mod compatibility](http://www.vivecraft.org/forge-mod-compatibility/), I'm sure you could gather more insight on the predicament.

**Comment by LuNeder** [comment]
Yeah, I too would like an option to use it without Optifine. Optifine breaks many great mods, such as Twilight Forest, and there are way better alternatives out there.

Would love an option to run Vivecraft with [Oculus (the mod, not the VR Gear)](https://www.curseforge.com/minecraft/mc-mods/oculus) + [Rubidium](https://www.curseforge.com/minecraft/mc-mods/rubidium) instead of Optifine.
